scribus/ui/fontcombo.cpp

```diff
@@ -320,3 +322,101 @@ void FontComboH::RebuildList(ScribusDoc *currentDoc, bool forAnnotation, bool fo
 	fontFamily->blockSignals(familySigBlocked);
 	fontStyle->blockSignals(styleSigBlocked);
 }
+
+fontFamilyDelegate::fontFamilyDelegate(QObject *parent):
+	QAbstractItemDelegate(parent)
+{
+}
+
+void fontFamilyDelegate::paint(QPainter *painter,
+							   const QStyleOptionViewItem &option,
+							   const QModelIndex &index) const
+{
+	QString text = index.data(Qt::DisplayRole).toString();
+	QFont font;
+	QStringList families = QFontDatabase().families();
+	QString name, style;
+	for (int i =0; i < families.size(); ++i)
+	{
+		name = families[i];
+		if (text.startsWith(name))
+		{
+			style = text;
+			style.remove(0,name.size()+1);
+			font = QFontDatabase().font(name, style, QFontInfo(option.font).pointSize());
+		}
+	}
+
+	font.setPointSize(QFontInfo(font).pointSize() * 3 / 2);
```

Comment:
in which cases scribus load non-system fonts? Maybe we just need  update Qfontdatabase, I am thinking though